src/test/java/PlacesClientTest.java

```diff
@@ -25,53 +23,50 @@
 
 /**
  * Test that PlacesClientImpl correctly converts a PlacesSearchResponse object obtained from the
- * Places API into a list of formatted addresses
+ * Places API into a list of Place IDs.
  */
 @RunWith(JUnit4.class)
-public class PlacesClientImplTest {
+public class PlacesClientTest {
```

Comment:
Should this be `PlacesClientImplTest`, because you are testing the static method in the impl file?
(With appropriate file name change?)

Reply:
Just moved this into PlacesResultUtility for consistency. Thanks for pointing that out!

Reply:
i.e. moved method to PlacesResultUtility and renamed this to PlacesResultUtilityTest

Reply:
`RESOLVED`